packages/angular/src/generators/setup-mf/files/ts-webpack/module-federation.config.ts__tmpl__

```diff
@@ -2,6 +2,18 @@ import { ModuleFederationConfig } from '@nx/webpack';
 
 const config: ModuleFederationConfig = {
   name: '<%= name %>',<% if(type === 'host') { %>
+  /**
+   * To use a remote that does not exist in your current Nx Workspace
+   * You can use the tuple-syntax to define your remote
+   *
+   * remotes: [['my-external-remote', 'https://nx-angular-remote.netlify.app']]
```

Comment:
Should it point to `remoteEntry.mjs` directly to better address the confusion/misunderstanding as in this issue https://github.com/nrwl/nx/issues/19841
```suggestion
   * remotes: [['my-external-remote', 'https://nx-angular-remote.netlify.app/remoteEntry.mjs']]
```

Reply:
Our utils should add the appropriate default file. I think it's fine to stick to that

Reply:
Strange, that is not what we observed after migrating to nx v17. Had to add remoteEntry.mjs suffix manually to all remotes on custom host.

Reply:
That’s interesting. I tried this out locally before I made the PR and it worked 🤔 id be interested in seeing a reproduction